README.md

```diff
@@ -191,3 +192,19 @@ be organised in an appropriate sub-folder there.
   device you need BYOD & 2FA set up.
 * A publish user with your email address with access to the organisation(s) you
   want to modify.
+
+### Mandatory requirements
+
+To successfully log into the system, you will need to:
+1. Create an account on DfE Sign-in
+   1. Get a DfE Sign-in admin to invite you (give them this link: 
+      https://signin-test-sup-as.azurewebsites.net/users)
+   2. Sign up from the email sent from DfE Sign-in
+2. Grant access to some providers: 
+   1. For local: `bundle exec bin/mcb users grant {email} {course_code}`
+   2. For `qa` and `production`: `bundle exec bin/mcb -E production users grant {email} {course_code}`
```

Comment:
prefer correct ordering

```
2. For `qa` and `production`:
  1. You will need to log into Azure first: `az login`
  2. `bundle exec bin/mcb -E {env} users grant {email} {provider_code}`
```
should be  {provider_code} vs {course_code}

Reply:
Apologies, what does `should be {provider_code} vs {course_code}` mean?

Reply:
didn't check the command...

but grant a user access to a provider I would think it needs provider code vs the course code
https://github.com/DFE-Digital/manage-courses-backend/blob/f517332bb30864e57946da82f940e68feb10e049/lib/mcb/commands/users/grant.rb#L4

so i guess command should be something like, 
`
`bundle exec bin/mcb -E {env} grant {email} {provider_code}`
`